Supernode1 (#1)
tomeshnet/toronto-community-network-docs: src/network/supernode1.md

# Supernode 1

![image](images/supernode1-view.jpg)

> Supernode 1 is located on top of 200 Woolner Ave, Toronto. The supernode was deployed along side a Cisco deployed infustructure belonging to the City of Toronto.

Reviewer: ```suggestion
Supernode 1 is located on top of 200 Woolner Ave, Toronto. The supernode was deployed along side a Cisco deployed infrastructure belonging to the City of Toronto.
```

> Supernode 1 is located on top of 200 Woolner Ave, Toronto. The supernode was deployed along side a Cisco deployed infrastructure belonging to the City of Toronto.

Reviewer 2: ```suggestion
Supernode 1 is located on top of 200 Woolner Ave, Toronto. The supernode was deployed alongside Cisco-deployed infrastructure belonging to the City of Toronto.
```

## Network

The supernode consists of 2 antennas and a router. The router is connected to the Cisco switch which is connected to the Bell 2000 modem that offers internet access for use as a gateway path.

The router acts as a Babel routing device. The antennas are configured in bridge mode.

![image](images/supernode1-network-layout.png)

## Hardware

### Antenna 1

> 

Reviewer 2: ```suggestion
```

> Antenna 1 is an Ubiquiti LAP-120 mounted on the West arm of the building, south side of the roof colocated with Cisco on their mounts. The antenna is facing south.

Reviewer: ```suggestion
Antenna 1 is an Ubiquiti LAP-120 mounted on the West arm of the building, on the south side of the roof, colocated with Cisco on their mounts. The antenna is facing south.
```

> Antenna 1 is an Ubiquiti LAP-120 mounted on the West arm of the building, on the south side of the roof, colocated with Cisco on their mounts. The antenna is facing south.

Reviewer 2: ```suggestion
Antenna 1 is an [Ubiquiti LAP-120](https://www.ui.com/airmax/liteap-ac/) mounted on the west arm of the building, on the south side of the roof, colocated with Cisco on their mounts. The antenna is facing south.
```

![image](images/supernode1-hardware-antenna1.jpg)

### Antenna 2

> Antenna 2 is an Ubiquiti LAP-120 mounted on the south arm of the building, east side of the roof colocated with Cisco on their mounts. The antenna is facing east.

Reviewer: ```suggestion
Antenna 2 is an Ubiquiti LAP-120 mounted on the south arm of the building, on the east side of the roof, colocated with Cisco on their mounts. The antenna is facing east.
```

![image](images/supernode1-hardware-antenna2.jpg)

### Router 1

> 

Reviewer 2: ```suggestion
```

> Router 1 is a Ubiquti EdgeRouter X-SFP mounted in the inside Cisco cabinet in the ballast room. It is connected to the exit node over a L2TP tunnel using UDP.

Reviewer: ```suggestion
Router 1 is a Ubiquti EdgeRouter X-SFP mounted inside the Cisco cabinet in the ballast room. It is connected to the exit node over a L2TP tunnel using UDP.
```

> Router 1 is a Ubiquti EdgeRouter X-SFP mounted inside the Cisco cabinet in the ballast room. It is connected to the exit node over a L2TP tunnel using UDP.

Reviewer 2: ```suggestion
Router 1 is an [Ubiquiti EdgeRouter X-SFP](https://www.ui.com/edgemax/edgerouter-x-sfp/) mounted inside the Cisco cabinet in the ballast room. It is connected to the exit node over a L2TP tunnel using UDP.
```

> Router is configured with a STATIC ip and only routes for the exit node and a secondary VPS server. Secondary VPS is used to provide an OpenVPN link for out of band management of the device.

Reviewer: ```suggestion
Router is configured with a static IP and only routes to the exit node and a secondary VPS server. The secondary VPS is used to provide an OpenVPN tunnel for secure remote management of the device.
```

> Router is configured with a static IP and only routes to the exit node and a secondary VPS server. The secondary VPS is used to provide an OpenVPN tunnel for secure remote management of the device.

Reviewer 2: Explain what is the `exit node` and which `secondary VPS server`, where is it and who manages it.

Author: Exit node will have its own page anyway. 

As for the VPS I'm not going to document this. Its a back door for in a temporary solution in case L2TP collapses in a unfinished exit node configuration.

![image](images/supernode1-hardware-cabinet2.jpg)

## Physical Environment

> Roof is accessable through ceiling hatches on the top floor of the building.

Reviewer: ```suggestion
The roof is accessible through ceiling hatches on the top floor of the building.
```

> ![image](images/supernode1-access-roof.jpg)

Reviewer: This image is confusing to look at, is it rotated? If you have a better image that would be helpful.

Author: It is taken  top down looking at the hatch from the roof. You can see the ladder on the right hand side.

I do not have any other images available.  There are several places where better images would be advised.

Reviewer: I have an image from inside the hatch, looking up, is that helpful? Otherwise we can leave it.

> The router is installed in a black metal cabinet located on the back wall of the ballast tank room on top of 200 Woolner. This room is accessable only from the roof. The entrance is on the east wall of the elevator hut. The doors are shorter then normal.

Reviewer: ```suggestion
The router is installed in a black metal cabinet, located on the back wall of the ballast tank room on the roof. This room is accessible only from the roof. The entrance is on the east wall of the elevator hut. The doors are shorter then normal.
```

Network cables are run through a hole in the east wall.

![image](images/supernode1-access-ballast.jpg)
![image](images/supernode1-hardware-cabinet.jpg)

> ## Neighborhood  Testing

Reviewer 2: ```suggestion
## Neighbourhood Testing
```

> Testing was done at several poits that where line of sight to the anteanna. Antenna was hand held not tune precisely. Results are for refrence only.

Reviewer: ```suggestion
Testing was done at several points that had line-of-sight to the antenna. The antenna was hand-held, not tuned precisely. Results are for reference only, and do not necessarily represent what a permanent deployment could attain at that distance.
```

> | Point     | Distance    | Ping      |  Speed RX/TX    | Signal   |

Reviewer: ```suggestion
| Point     | Distance (m)    | Ping      |  Speed RX/TX (mbps)   | Signal (dBm)   |
```
If you put the units here they can be removed from all the cells below, it might make the table cleaner.

|-----------|-------------|-----------|-----------------|----------|
| a         |   200m      |    4.57   |  86/194Mbps     | -64      |
| b         |   200m      |    2.86   |  80/194Mbps     | -60      |
| c         |   237m      |    5.82   |   50/178Mbps    | -58      |
| d         |   270m      |    5.27   |  103/149Mbps    | -50      |
| e (1)     |   396m      |     5.11  |  83/172Mbps     | -58      |
| f         |   2520m     |   4.57    |  8.68/1.6Mbps   | -81      |
| f (2)     |   2520m     |    4.57   |  64.58/40.79Mbps| -74      |

(1) Used both Loco AC and LiteBeam AC. Both performed the same.  
(2) Second attempt done after corrected issue with sn1a2.

## Additional notes

> EdgeRouter X-SFP does not accelerate traffic over tunnels. Due to this, speeds are currently limited to around 400Mbps to exit node.

Reviewer: ```suggestion
The EdgeRouter X-SFP does not accelerate traffic over tunnels. Due to this, speeds to the exit node are currently limited to around 400Mbps.
```

> (1) Used both Loco AC and LiteBeam AC. Both performed the same.  

Reviewer 2: Link these, use exact names.

Author: Updated

> (2) Second attempt done after corrected issue with sn1a2.

Reviewer 2: ```suggestion
(2) Second attempt done after correcting issue with `sn1a2`.
```

> ## Additional notes

Reviewer 2: ```suggestion
## Additional Notes
```
